moda-contracts/src/interfaces/ICatalog.sol

```diff
@@ -0,0 +1,336 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+/**
+ * @title IRegistry
+ * @dev Interface for Registry
+ */
+interface ICatalog {
+    /**
+     * @dev Status of a registered track
+     */
+    enum TrackStatus {
+        PENDING,
+        VALIDATED,
+        INVALIDATED
+    }
+
+    /**
+     * @dev Represents a registered track
+     * TrackStatus - The status of the track (pending, validated, invalidated)
+     * TrackOwner - The address that registered the track
```

Comment:
The track owner is not necessarily the one that registers it. It could be a manager.

Maybe something like this?
`The account that owns the rights to a track. This is usually the artist address, but could be a contract.`

Reply:
I think I will just have the artist address recorded here, regardless of which address is registering the track, it is only the artist address that is used to run all the checks.

Reply:
Hmm yeah true maybe artist is too specific? should it be something like trackRightsHolder?

Reply:
Im leaving as trackArtist for now but maybe trackRightsHolder should replace all uses of the word artist?

Reply:
Yeah, good call. Let's keep it more general

Reply:
Yeah, I think `trackRightsHolder` is a better name

Reply:
This is a good question for Sean. I'm on the fence. I think either work